src/client/components/FooterImprovement/FooterImprovement.styles.css

```diff
@@ -0,0 +1,25 @@
+footer {
+  background-color: #f8f8f8;
+  position: fixed;
+  width: 100%;
+  left: 0px;
```

Comment:
nice work Kiran . here if you also add "bottom :0" , then you insure the footer will stay  in the bottom of window even if there are no other elements in the page